server/capabilities.go

```diff
@@ -142,6 +176,13 @@ func (c *Configuration) EnrichCapabilities(host string) (*paymail.CapabilitiesPa
 	for key, cap := range c.callableCapabilities {
 		payload.Capabilities[key] = serviceUrl + string(cap.Path)
 	}
+	for key, cap := range c.nestedCapabilities {
+		payload.Capabilities[key] = make(map[string]interface{})
+		for nestedKey, nestedCap := range cap {
+			payload.Capabilities[key].(map[string]interface{})[nestedKey] = serviceUrl + string(nestedCap.Path)
```

Comment:
It might be safer if we checked whether or not the type assertion works.
```golang
nastedObject, ok := payload.Capabilities[key].(map[string]interface{})
if !ok {
    //some warning or even error returning
    continue
}
nestedObject[nestedKey] = serviceUrl + string(nestedCap.Path)
```